Execute.vbs

```diff
@@ -0,0 +1,3 @@
+Set WshShell = CreateObject("WScript.Shell" ) 
```

Comment:
Lets remove the *.vbs file from the project. Any kind of execution helper should be thought only from a build system perspective.